restapi/stories.go

```diff
@@ -0,0 +1,37 @@
+package restapi
+
+import (
+	"net/http"
+
+	"github.com/go-chi/chi"
+	"github.com/go-chi/render"
+	"github.com/uwblueprint/shoe-project/internal/database/models"
+	"github.com/uwblueprint/shoe-project/restapi/rest"
+	"gorm.io/gorm"
+)
+
+func (api api) ReturnAllStories(w http.ResponseWriter, r *http.Request) render.Renderer {
+	var stories []models.Story
+	err := api.database.Find(&stories).Error
+
```

Comment:
usually in go codebases the line generating the error and error check are joined together and there is no space